openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -118,22 +118,32 @@
 - control_key: AU-4
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: partial
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
+        OpenShift does not control the storage capacity for the
+        OpenShift API audit logs when they are generated.
 
-        https://issues.redhat.com/browse/CMP-159
+        Either the OpenShift API audit logs need to be consumed
+        by an external SEIM and the storage capacity managed there
+        as per organization policy requirments or the operating system on
+        which OpenShift is running needs to have /var/log/audit storage
+        capacity configured to meet organization policy requirments.
 
 - control_key: AU-4 (1)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
+        OpenShift does not control the storage capacity for the
+        OpenShift API audit logs when they are generated.
 
-        https://issues.redhat.com/browse/CMP-160
+        Either the OpenShift API audit logs need to be consumed
+        by an external SEIM and the storage medium managed there
+        as per organization policy requirments or the operating system on
+        which OpenShift is running needs to have /var/log/audit storage
+        medium configured to meet organization policy requirments.
```

Comment:
Note AU-4(1) requires off-loading audit records. Where can documentation be found on how to offload all OCP4 audit records?

Reply:
This should be coming from RHCOS and/or RHEL documentation and not OCP documentation.  Therefore, this section of the security controls for those components should have those links.

Reply:
> This should be coming from RHCOS and/or RHEL documentation and not OCP documentation. Therefore, this section of the security controls for those components should have those links.

OpenShift and CoreOS logging are separate. OpenShift will need its own configuration to forward logs, eg through a ``Log Forwarding Custom Resource``. Much of that now appears documented here:

https://docs.openshift.com/container-platform/4.3/logging/config/cluster-logging-log-forwarding.html

Perhaps some of that language can be used?